src/libostree/ostree-repo-private.h

```diff
@@ -55,6 +55,9 @@ G_BEGIN_DECLS
  * in a summary file. */
 #define OSTREE_COMMIT_TIMESTAMP "ostree.commit.timestamp"
 
+/* Well-known keys for the commit metadata */
+#define OSTREE_REF_COLLECTION_BINDING "ostree.ref-collection-binding"
```

Comment:
As discussed in issue #983, it looks like this needs to be split up into `ostree.ref-binding` (type `as`) and `ostree.collection-binding` (type `s`), with `ostree.collection-binding` only being used if `OSTREE_ENABLE_EXPERIMENTAL_API` is defined.